src/components/PageSections/Menu/ThemesTopicsMenu.js

```diff
@@ -31,7 +31,11 @@ const TopicsLinks = props =>
 
 const ThemesTopicsMenu = props => {
   return (
-    <nav className="coa-ThemesTopicsMenu">
+    <nav
+      className={`coa-ThemesTopicsMenu ${
```

Comment:
A note on what extraClasses is or intended use would be cool, though I can see from the footerSiteMap what the intended use is.